### PR TITLE
#20 Implements groupby and distinct

### PR DESCRIPTION
**Pull request by ayz4sci**
For https://github.com/afollestad/inquiry/issues/20

### CODE REVIEW

**Comment by HeinrichReimer** [comment]
Codacy reports some issues with your PR.
Please check the [results](https://www.codacy.com/app/drummeraidan_50/inquiry/pullRequest?prid=544669) and update the PR :)

File: library/src/main/java/com/afollestad/inquiry/ClassRowConverter.java [hunk 1]
```diff
@@ -23,6 +24,7 @@
 import java.io.Serializable;
 import java.lang.reflect.Field;
 import java.util.ArrayList;
+import java.util.Arrays;
```
Comment (HeinrichReimer) [inline library/src/main/java/com/afollestad/inquiry/ClassRowConverter.java]: Import not needed.

File: library/src/main/java/com/afollestad/inquiry/Query.java [hunk 2]
```diff
@@ -316,6 +337,9 @@ else if (mInquiry.mContext == null)
         if (mProjection == null)
             mProjection = ClassRowConverter.generateProjection(mRowClass);
 
+        if (mDistinct != null)
```
Comment (HeinrichReimer) [inline library/src/main/java/com/afollestad/inquiry/Query.java]: Matter of personal preference but probably having braces (`{}`) wouldn't be bad.

**Comment by afollestad** [comment]
Merge conflicts will have to be resolved, also some tests will be needed.

**Pull request closed without merging**